qiskit/pulse/macros.py

```diff
@@ -17,11 +17,28 @@
 from qiskit.pulse import channels, exceptions, instructions, utils
 from qiskit.pulse.instruction_schedule_map import InstructionScheduleMap
 from qiskit.pulse.schedule import Schedule
+from qiskit.transpiler.target import Target
+from qiskit.utils.deprecation import deprecate_arg
```

Comment:
It seems that this import statement causes a circular import, so we need to resolve it.